Doc/library/http.client.rst

```diff
@@ -115,6 +115,25 @@ The module provides the following classes:
       The *strict* parameter was removed. HTTP 0.9 style "Simple Responses" are
       no longer supported.
 
+This module provides the following function:
+
+.. function:: parse_headers(fp)
+
+   Parse the headers from a file pointer *fp* representing a HTTP
+   request/response. The file has to be a :class:`BufferedIOBase` reader
+   (i.e. not text) and must provide a valid :rfc:`2822` style header.
+
+   This function returns an instance of :class:`email.message.Message`
+   that holds the header fields, but no payload
+   (the same as :attr:`HTTPResponse.msg`
+   and :attr:`http.server.BaseHTTPRequestHandler.headers`).
+   It returns with the file pointer *fp* ready to read the HTTP body.
```

Comment:
I've read this a few times and find myself getting slightly tripped up. What about wording this line as `After returning, the file pointer *fp* is ready to read the HTTP body.`?

The way it's written makes me think it's going to return `fp`, although it mentions above that it's returning a `Message`. I think it may be more clear to separate the fact that the function is going to return one thing (`Message`) and another thing (`fp`) will be ready to read.